portlayer/exec/container.go

```diff
@@ -176,3 +176,23 @@ func (c *Container) Start(ctx context.Context) error {
 
 	return nil
 }
+
+func (c *Container) Stop(ctx context.Context) error {
+	defer trace.End(trace.Begin("Container.Stop"))
+
+	if c.vm == nil {
+		return fmt.Errorf("vm not set")
+	}
+
+	//TODO: make the shutdown much cleaner, right now we just pull the plug on the vm.(may need corresponding work in tether.)
+
+	// Power off
+	_, err := tasks.WaitForResult(ctx, func(ctx context.Context) (tasks.ResultWaiter, error) {
+		return c.vm.PowerOff(ctx)
```

Comment:
Eventually yes, since the docker documentation mentions that the `docker stop` sends a sigterm and then a sigkill we will eventually want to implement something similar in the tether. For minimal functionality PowerOff is being used.